share/csm_share/shr/shr_frz_mod.F90.in

```diff
@@ -6,7 +6,7 @@
 !===============================================================================
 
    use shr_kind_mod, only: R8=>SHR_KIND_R8, CS=>SHR_KIND_CS
-   use shr_log_mod, only: s_logunit => shr_log_Unit
+   use shr_log_mod, only: s_logunit => shr_log_Unit, shr_log_level
```

Comment:
shr_log_level is 1 on the component root node and 0 on all other nodes - this reduces noisy output to log files.